patternfly/patternfly-elements
# provide explicit global variable names to rollup

This PR removes the following warning when building elements.

    (!) Missing global variable name
    Use output.globals to specify browser global variable names corresponding to external modules
    /home/mclayton/projects/pfe/elements/pfelement/dist/pfelement.umd (guessing 'PFElement')

It does so by using `pfelement.className` from each element's package.json to determine what the global variable name should be.

### Testing instructions

```
git checkout fix-missing-global-variable-warning
npm run build
```

In the build log, you should not see any `(!) Missing global variable name` warnings.  You can verify this more easily with grep.

```
git checkout fix-missing-global-variable-warning
npm run build 2>&1 | grep "Missing global"
```
That command should run for a while, but should not have any output.

For reference, when that command is run on `master` the output looks like the following:

```
$ npm run build 2>&1 | grep "Missing global"
@patternfly/pfe-icon-panel: (!) Missing global variable name
@patternfly/pfe-badge: (!) Missing global variable name
@patternfly/pfe-select: (!) Missing global variable name
@patternfly/pfe-avatar: (!) Missing global variable name
@patternfly/pfe-health-index: (!) Missing global variable name
@patternfly/pfe-datetime: (!) Missing global variable name
@patternfly/pfe-progress-indicator: (!) Missing global variable name
@patternfly/pfe-content-set: (!) Missing global variable names
@patternfly/pfe-page-status: (!) Missing global variable name
@patternfly/pfe-collapse: (!) Missing global variable name
@patternfly/pfe-icon-panel: (!) Missing global variable name
@patternfly/pfe-icon: (!) Missing global variable name
@patternfly/pfe-autocomplete: (!) Missing global variable name
@patternfly/pfe-cta: (!) Missing global variable name
@patternfly/pfe-card: (!) Missing global variable name
@patternfly/pfe-toast: (!) Missing global variable name
@patternfly/pfe-modal: (!) Missing global variable name
@patternfly/pfe-band: (!) Missing global variable name
@patternfly/pfe-badge: (!) Missing global variable name
@patternfly/pfe-markdown: (!) Missing global variable name
@patternfly/pfe-health-index: (!) Missing global variable name
@patternfly/pfe-progress-indicator: (!) Missing global variable name
@patternfly/pfe-datetime: (!) Missing global variable name
@patternfly/pfe-select: (!) Missing global variable name
@patternfly/pfe-page-status: (!) Missing global variable name
@patternfly/pfe-tabs: (!) Missing global variable name
@patternfly/pfe-content-set: (!) Missing global variable names
@patternfly/pfe-accordion: (!) Missing global variable name
@patternfly/pfe-number: (!) Missing global variable name
@patternfly/pfe-cta: (!) Missing global variable name
@patternfly/pfe-toast: (!) Missing global variable name
@patternfly/pfe-card: (!) Missing global variable name
@patternfly/pfe-modal: (!) Missing global variable name
@patternfly/pfe-collapse: (!) Missing global variable name
@patternfly/pfe-avatar: (!) Missing global variable name
@patternfly/pfe-icon: (!) Missing global variable name
@patternfly/pfe-band: (!) Missing global variable name
@patternfly/pfe-autocomplete: (!) Missing global variable name
@patternfly/pfe-navigation: (!) Missing global variable names
@patternfly/pfe-tabs: (!) Missing global variable name
@patternfly/pfe-accordion: (!) Missing global variable name
@patternfly/pfe-number: (!) Missing global variable name
@patternfly/pfe-markdown: (!) Missing global variable name
@patternfly/pfe-navigation: (!) Missing global variable names

```
